manifests/autotune-configs/hotspot-micrometer-config.yaml

```diff
@@ -39,7 +39,8 @@ tunables:
   - response_time
   - throughput
 
-- name: gc
+# Enable zGC if Java >=17
+- name: GCPolicy
```

Comment:
This is currently used by the code to set the GC string and needs to be "gc"

Reply:
updated.